Update images to reflect the latest UI changes. (#71)
wso2/docs-choreo-dev: en/docs/services/create-your-first-service.md

# Create Your First Service

To learn how to create a service, let's try a simple scenario. In this example, you are creating a service that consumes a public API that fetches the current active COVID-19 cases in a specified country, testing it, deploying it, and then observing its performance.

## Step 1: Create the service

To create the service, follow the procedure below: 

1. Access the Choreo Development Console via https://console.choreo.dev/.
    
2. Sign in using either your Google or GitHub credentials.
    

> 3. In the **Services** page, click **Create**. You are directed to the low-code view

Reviewer: ```suggestion
3. In the **Services** page, click **Create**. You are directed to the low-code view.
```

    ![Resource Configuration](../assets/img/services/configure-api-trigger.png)
    
    Then click **Save API**. Now you have saved the API trigger that starts your service.
    
6. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.

    1. Click **HTTP**.
    
        ![Select HTTP Connection](../assets/img/services/select-http-connection.png)
        
    2. Enter information as follows:
    
        | **Field**           | **Value**                    |
        |---------------------|------------------------------|
        | **Connection Name** | `httpEndpoint`               |
        | **URL**             | `"https://api.covid19api.com"` |
        
        Click **Save**.
        
    3. Click the **+** icon below the HTTP API call you added. 
    
        ![Update Existing Connection](../assets/img/services/update-existing-connection.png)
        
    4. Click **API Calls**, and then click on the existing connection.
    
        ![Select Existing connection](../assets/img/services/select-existing-connection.png)
        

>         Then enter information as follows:

Reviewer: Shouldn't this be the number `5`?

Author: Changed as mentioned.

4. In the **Configure API Trigger** form, select **GET** as the HTTP method, and enter `activecases` as the relative path from host.

    ![Resource Configuration](/en/assets/img/services/configure-api-trigger.png)
    
    Then click **Save API**. Now you have saved the API trigger that starts your application.
    
5. Click **API Calls**, and then enter information as follows to configure the HTTP connection that connects the public API used in this scenario to the API trigger you configured.

    1. Click **HTTP**.
    
        ![Select HTTP Connection](/en/assets/img/services/select-http-connection.png)
        
    2. Enter information as follows:
    
        | **Field**           | **Value**                    |
        |---------------------|------------------------------|
        | **Connection Name** | `httpEndpoint`               |
        | **URL**             | `https://api.covid19api.com` |
        
        Click **Save**.
        
    3. Click the **+** icon below the HTTP API call you added. 
    

>         ![Update Existing Connection](/en/assets/img/services/update-existing-connection.png)

Reviewer: Use relative paths. Check all the places. Changes were added in https://github.com/wso2/docs-choreo-dev/pull/63/files

Author: Corrected

2. To deploy the service, click **Deploy**.

    The status of the service changes to **Deployed**, and the following logs appear to indicate that the product is successfully deployed.    

    ![Deployment Logs](../assets/img/services/deployment-logs.png)

> 3. Once the service is deployed, click on the **Go Live** tab below the low-code design view. Then click **://cURL** and copy the cURL command that is displayed.

Reviewer: ```suggestion
3. Once the service is deployed, click on the **Go Live** tab below the low-code design view. Then click **://cURL** icon and copy the cURL command that is displayed.
```

Author: Isn't it a button and not an icon? I did not use the word "button" because the Microsoft Style guide specifies to avoid using it when it is clear enough when we give the button text.

Reviewer: ack.

    
    ![Execution History](../assets/img/services/execution-history.png)
    
To observe the `covid-stats` service by checking the statistics generated as a result of the cURL commands you issued, proceed to Step 4. 

## Step 4: Observe the service

To observe the `covid-stats` service, click the **Observe** icon in the left panel.

![Observe Icon](../assets/img/services/observe-icon.png)

The throughput and the latency of the `covid-stats` service are visualized as follows:

![Visualization of Throughput and Latency](../assets/img/services/visualization-of-statistics.png)

> Congratulations! Now you have successfully created a services, tested it, deployed it, and obnserved its statistics.

Reviewer: ```suggestion
Congratulations! Now you have successfully created a service, tested it, deployed it, and observed its statistics.
```
